### ISSUE TITLE
Direction of the Binder project

### ISSUE THREAD

**Issue by betatim** [jupyterhub/team-compass#136]
Where should (and can) Binder go next? How do we do it?

These are some of the big questions that we need to answer in this thread.

To help us do this the first part of this will be reading + privately answering some key questions. The reading material should state facts about the status quo and give background on the questions we want to answer. It should not be an exhaustive and complete history of the project :)

In a second step we bring the answers from the individual people together to make one out of them. The point of first doing it in private is that it forces each of us to have our own independent ideas, otherwise we might get stuck on one particular aspect that just happens to get suggested first.

We have started a hackmd with [background, context and questions](https://hackmd.io/XzWqI4laQZagxFFRqZGN-A?edit). It isn't finished so please do add to it. I would propose we take till Friday end of business in California to work on the HackMD. Hold back on answering the questions till we know what they are.

**Comment by betatim** [comment]
Friday California time has come and gone. Please take a moment to jot down your answers to the questions that ended up being in the HackMD. Please don't post them in the hackmd just yet so that we can get independent thoughts from everyone. Writing down your answers shouldn't take a huge amount of time, think breakout session at the team meeting, not school essay. You have 72h from now before we move on to a phase of consolidating all the ideas into one coherent plan.

We have a conversation with Moore coming up in a bit where not everyone can/needs to be present. If you want your point of view to be considered writing down your answers to this is your way of doing that.

**Comment by choldgraf** [comment]
I haven't had the cycles to do this because of strata SF, I can try to get to this soon.

**Comment by choldgraf** [comment]
ok I filled it out! @betatim perhaps the team should be pinged on this one, since we have discovered that many don't "follow" this repository (yet)?

**Comment by betatim** [comment]
Several people have written down their ideas already. If you have thoughts but haven't shared them on the HackMD yet please do by Monday evening California time.

I propose after that time passes everyone reads what everyone else wrote.

For making one coherent version out of it I think having one person write that summary is a good idea. I don't think as a committee we can write one coherent text (too many different styles, etc). Once there is a proposed text we can discuss it and edit it with PRs/suggestions in a google doc. I think going through one person helps keep the document consistent (as opposed to everyone editing willy nilly).

I'm wondering: should we leave some time between Monday evening and starting to write one summary to discuss things?

TL;DR: what do you think of "single editor" and "leave time for questions?

**Comment by betatim** [comment]
https://discourse.jupyter.org/t/creating-a-specification-for-reproducible-repositories/569 this is a thread worth having a look at in terms of "future directions" for the project.

**Comment by betatim** [comment]
Is there someone who would like to take on the role of being the "single editor"? It doesn't mean you have to decide or that you get to decide the direction of the project. You do get to decide how we say it, keep us "on message" in the writing and all that.

**Comment by choldgraf** [comment]
I'm happy to do this, though also will defer to anybody else if they'd like to give a stab at it.

**Comment by betatim** [comment]
Sounds like a plan. There don't seem to be any questions that need to be addressed before making a first draft.

I would propose we move forward with you as editor and look forward reading it. As your old supervisor used to say: remove all the extra words ;)

**Comment by willingc** [comment]
I'm going to add some thoughts now that I'm on the other side of the language creators event.

**Comment by choldgraf** [comment]
Cool! I will hold off on any summarizing for a bit then. And congrats on getting through the event Carol! 😊

**Comment by willingc** [comment]
@choldgraf Added my thoughts to the hackpad

**Comment by choldgraf** [comment]
hey all - here is a quick summary of each of the questions that folks responded to. I didn't have a ton of time to really do a big deep dive, but hopefully this captures many of the main points. Please feel free to edit/add/remove/comment/etc. My goal was to capture what others had said, rather than be prescriptive about what we "should" do.

https://hackmd.io/XzWqI4laQZagxFFRqZGN-A?both#Short-summary

**Comment by betatim** [comment]
Nice work. Reading now.

I think overall we should aim for
* no repeated points, merge them instead
* somehow sort them by amount of "support"
* (In the context of the conversation with Moore maybe decide to park some of the ideas that aren't at the top of the list in order to focus and polish things a bit)

WDYT?

**Comment by choldgraf** [comment]
makes sense to me - though many of them aren't specific enough for a kind of "laundry list" to include in a grant, I feel. What do you think? I agree with your methodology for culling down the items in each section.
